common/djangoapps/student/admin.py

```diff
@@ -316,6 +321,94 @@ class Meta(object):
         model = CourseEnrollmentAllowed
 
 
+@admin.register(LoginFailures)
+class LoginFailuresAdmin(admin.ModelAdmin):
+    """Admin interface for the LoginFailures model. """
+    list_display = ('user', 'failure_count', 'lockout_until')
+    raw_id_fields = ('user',)
+    search_fields = ('user__username', 'user__email')
+    actions = ['unlock_student_accounts']
+    change_form_template = 'admin/student/loginfailures/change_form_template.html'
+
+    def has_pem(self, request, method):
```

Comment:
Sure, that could work. I was trying to follow the same patter [used before](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/admin.py#L219) but instead of doing a `getattr` it's more explicit sending the actual callable.
The only thing about using a decorator is where to put it since it's very specific to this class and defining a decorator inside a class would read quite odd.
What do you think?